container-images/ramalama/latest/Containerfile

```diff
@@ -1,5 +1,14 @@
 FROM registry.access.redhat.com/ubi9/ubi:9.4
 
+# renovate: datasource=github-releases depName=huggingface/huggingface_hub extractVersion=^v(?<version>.*)
+ARG HUGGINGFACE_HUB_VERSION=0.24.2
```

Comment:
Renovate will go to the huggingface/huggingface_hub GitHub releases and grab the "latest" (non-prerelease) version.

Python packages do not contain the `v` in the version number, whereas the GitHub releases do.  We need to tell Renovate to ignore the `v` in the version string.